src/thunderbots/software/ai/main.cpp

```diff
@@ -32,14 +32,30 @@ void worldUpdateCallback(const thunderbots_msgs::World::ConstPtr &msg)
 {
     thunderbots_msgs::World world_msg = *msg;
     World new_world = Util::ROSMessages::createWorldFromROSMessage(world_msg);
-    world.updateBallState(new_world.ball());
-    world.updateFieldGeometry(new_world.field());
-    world.updateEnemyTeamState(new_world.enemyTeam());
-    world.updateFriendlyTeamState(new_world.friendlyTeam());
-    world.updateTimestamp(new_world.getMostRecentTimestamp());
-    RefboxGameState new_game_state =
-        Util::ROSMessages::createGameStateFromROSMessage(world_msg.refbox_data.command);
-    world.updateRefboxGameState(new_game_state);
+    // During RoboCup 2019, we saw weird cases where vision frames wold randomly appear
+    // mirrored. This filter checks if our goalie appears in the enemy defense area to
+    // detect mirroring, and ignores the data if mirroring is detected
+    if (!Util::DynamicParameters::AI::vision_flipping_filter.value() ||
```

Comment:
Could we change `vision_flipping_filter` to `vision_flipping_filter_enabled`?

Reply:
done